backend/src/main/java/de/bund/digitalservice/ris/norms/application/service/PublishService.java

```diff
@@ -85,38 +88,59 @@ public void processQueuedFilesForPublish() {
       }
     });
 
-    loadLastMigrationLogPort
+    var lastMigrationLog = loadLastMigrationLogPort
```

Comment:
This filtering looks too complex, why not just:
`var lastMigrationLog = loadLastMigrationLogPort.loadLastMigrationLog().filter(f -> !f.isCompleted());`
Is it just for the logging? I think we dont need the info when a migration log was found that was already completed right?